docs/buildah-commit.md

```diff
@@ -10,8 +10,7 @@ buildah\-commit - Create an image from a working container.
 Writes a new image using the specified container's read-write layer and if it
 is based on an image, the layers of that image.  If *image* does not begin
 with a registry name component, `localhost` will be added to the name.  If
-*image* is not provided, the values for the `REPOSITORY` and `TAG` values of
-the created image will each be set to `<none>`.
+*image* is not provided, the image will have no name.
```

Comment:
we've had a lot of confusion over the  '<none>' tags over the years.  Would you mind adding :
```suggestion
*image* is not provided, the image will have no name, denoted by the value `<none>` in the REPOSITORY and TAG values of the created image.
```

Reply:
That text suggests to me that the text `<none>` is actually stored somewhere related to the image, which is not the case.

Reply:
Can we state that this will be listed this way when executing buildah images.

Reply:
OK,  I understand the concern @nalind, how's about something like:

```suggestion
*image* is not provided, the image will have no name.  When an image has no name, the `buildah images` command will display `<none>` in the REPOSITORY and TAG columns. 
```
I know it's kind of cross command man paging, but it's been a confusion point.  I'm fine if you want to change the above wording.

Reply:
Sure, but we also need to add `buildah-images(1)` man page to the bottom.